app/assets/javascripts/covid19/src/data/Covid19InfoFromJsonRepository.ts

```diff
@@ -238,8 +238,6 @@ function getPdbValidations(pdb: Data.Pdb): PdbValidation[] {
                           badgeColor: "w3-blue",
                           queryLink: refModel.queryLink,
                       };
-                  default:
-                      return undefined;
               }
```

Comment:
Now that we've removed the default case, eslint is not happy (rule: "array-callback-return"). Since TS already backs us up with this, it's better just to disable the rul: `"array-callback-return": "off",`